.rubocop_todo.yml

```diff
@@ -440,6 +440,8 @@ Style/AccessorMethodName:
 Style/BracesAroundHashParameters:
   Exclude:
     - 'app/models/concerns/searchable.rb'
+    - 'spec/features/budgets/investments_spec.rb'
```

Comment:
This two where added since we didn't agree on the usage of curly braces on those two specs. Will re-evaluate later as we tackle again rubocop_todo list